types/react-router-navigation/react-router-navigation-tests.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from "react";
 import { Text } from "react-native";
-import { NativeRouter, Link } from "react-router-native";
```

Comment:
this is a tests-only dependency, which DT maintainers have said is bad practice in the past.

rather than introducing a production dependency to make this work (one the actual types don't use), i've added some mock interfaces/functions since the shape doesn't actually matter for testing this as long as the parent accepts children.

Reply:
thanks for cleaning up